invenio_app_ils/ui/src/invenio_app_ils/common/components/ResultsTable/__tests__/ResultsTable.js

```diff
@@ -181,23 +171,16 @@ describe('ResultsTable tests', () => {
       <ResultsTable
         rows={results}
         showMaxRows={1}
-        history={historyFn}
         seeAllComponent={buttonObj()}
         rowActionClickHandler={() => {}}
       />
     );
 
     const button = component.find('TableFooter').find('button');
     button.simulate('click');
-    expect(mockedHistoryPush).toHaveBeenCalled();
```

Comment:
Shouldn't we still keep these asserts to make sure the page actually changed? This same applies to the other test files too.